java/client/src/main/java/glide/api/models/commands/ZaddOptions.java

```diff
@@ -0,0 +1,91 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.SortedSetBaseCommands;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Optional arguments to {@link SortedSetBaseCommands#zadd(String, Map, ZaddOptions, boolean)},
+ * {@link SortedSetBaseCommands#zadd(String, Map, ZaddOptions)} and {@link
+ * SortedSetBaseCommands#zaddIncr(String, String, double, ZaddOptions)}
+ *
+ * @see <a href="https://redis.io/commands/zadd/">redis.io</a>
+ */
+@Builder
+public final class ZaddOptions {
+    /**
+     * Defines conditions for updating or adding elements with {@link SortedSetBaseCommands#zadd}
+     * command.
+     */
+    private final ConditionalChange conditionalChange;
+
+    /**
+     * Specifies conditions for updating scores with zadd {@link SortedSetBaseCommands#zadd} command.
+     */
+    private final UpdateOptions updateOptions;
+
+    @RequiredArgsConstructor
+    @Getter
```

Comment:
Do you use these getters? If not, probably you are missing a test. If you still don't use them -delete.

Reply:
Good point. We can probably delete them.